Examples/Swift/ViewController.swift

```diff
@@ -298,21 +302,32 @@ class ViewController: UIViewController, MGLMapViewDelegate, CLLocationManagerDel
     }
     
     func setupMapView() {
-        guard mapView == nil else { return }
-        mapView = NavigationMapView(frame: view.bounds)
+        guard self.mapView == nil else { return }
+        let mapView = NavigationMapView(frame: view.bounds)
+        self.mapView = mapView
+        
+        mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.delegate = self
+        mapView.navigationMapDelegate = self
+        mapView.userTrackingMode = .follow
+        let bottomPadding = (view.frame.height + view.frame.origin.y) - bottomBar.frame.origin.y
         
-        mapView!.autoresizingMask = [.flexibleWidth, .flexibleHeight]
-        mapView!.delegate = self
-        mapView!.navigationMapDelegate = self
-        mapView!.userTrackingMode = .follow
+        var topPadding: CGFloat = 0.0
+        if #available(iOS 11.0, *) {
+            topPadding = view.safeAreaInsets.top
+        } else if let navCon = navigationController {
+            topPadding = navCon.navigationBar.frame.size.height
+        }
+        mapView.contentInset = UIEdgeInsets(top: topPadding + 25, left: 25, bottom: bottomPadding + 25, right: 25)
```

Comment:
Could we assign this `25` value to a variable? This would make it more self-documenting. Seems like a magic number.